mentat/code_file_manager.py

```diff
@@ -102,7 +102,7 @@ async def write_changes_to_files(
                 continue
 
             if not file_edit.is_creation:
-                stored_lines = self.file_lines[file_edit.file_path]
+                stored_lines = self.get_file_lines(file_edit)
```

Comment:
Hmm, file edit's aren't supposed to have relative paths (it should always be absolute); how do I reproduce the problem? I think we should fix the actual problem (file edit not having absolute paths) rather than throwing this bandaid on it

Reply:
I updated the description with two ways to reproduce. They are absolute. But they also have '..''s in them.

Reply:
e.g. I get errors like: `KeyError: PosixPath('/home/jake/Development/mentat/../../dotfiles/zsh/zshrc')`

Reply:
Oh yeah, I'll do that. It'll probably fix another bug I've noticed that isn't fixed by this: the line numbers aren't displayed correctly while streaming:
![image](https://github.com/AbanteAI/mentat/assets/385534/5da7f530-4a3f-4516-b76b-5d490c7465ff)

Reply:
Display info also fixed now.